p2p/state.py

```diff
@@ -238,18 +267,19 @@ def msg_queue_maxsize(self) -> int:
 
     async def _periodically_report_progress(self) -> None:
         while self.is_running:
+            requested_nodes = sum(
+                len(node_keys) for _, node_keys in self._requested_nodes.values())
             self.logger.info("====== State sync progress ========")
```

Comment:
Whatcha think about going ahead and re-structuring this output to be a single line similar to geth's?

```
processed: %11d  tnps: %5s committed: %11d scheduled: %7d timeout: %10d duplicates: %11d
```

Reply:
Sure, but what's `tnps` and why the different formatting lengths for every item?

Reply:
`tnps -> tree-nodes-per-second`, the numbers are the fixed width spacing so that each line is consistently easy to read (and I tried to pick numbers appropriate for the expected scale of the values we expect to see in those places)